### PR TITLE
ng-route replaced with ui-route

### PR DESCRIPTION
**Pull request by AhmedHassan07**
ui-router is added in place of ng-route. It is helpful for state base routing.

### CODE REVIEW

**Comment by chaudhryjunaid** [comment]
Seems like awesome work Ahmed! Would review and merge soon! 👍

**Comment by AhmedHassan07** [comment]
Html 5 mode is turned on.
Sign in, signup and 404 pages are on angular end.There is a little bit of design issue on sign in and sig up pages i.e fields are below the social login icons.i will fix it soon. and also i have removed click on social login icon because we need social authentication on angular side. i will do this task up next.

File: config/express.js [hunk 1]
```diff
@@ -76,6 +76,10 @@ module.exports = function(app, passport) {
       require(path.resolve(routePath))(app);
     });
 
+    app.get('*',  function (req, res, next) {
+            res.render('index');
+    });
+
     app.use('*',function(req, res){
```
Comment (chaudhryjunaid) [inline config/express.js]: @AhmedHassan07 This should be removed then?

**Comment by chaudhryjunaid** [comment]
Overall pretty cool work @AhmedHassan07! Just one quick comment, deleting an article and then going to the article list for the first time shows the article still present but on refresh the deleted article disappears. This should disappear the first time. Can you look into fixing this?

**Comment by chaudhryjunaid** [comment]
LGTM!

**Comment by AhmedHassan07** [comment]
Thanks,
 Some changes i have pushed i.e 404 routing removed from server.and some of
article routes were changed from articles to article because of conflict of
routes on server side.
I was also checking that bug you reported about article deleting. I was not
able to reproduce it, i will fix this issue ASAP i reproduce this issue.

**Pull request merged**